pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -32,3 +32,17 @@
 TYPE_VALUES = {TYPE_STANDARDS, TYPE_INFO, TYPE_PROCESS}
 # Active PEPs can only be for Informational or Process PEPs.
 ACTIVE_ALLOWED = {TYPE_PROCESS, TYPE_INFO}
+
+# map of topic -> additional description
+SUBINDICES_BY_TOPIC = {
+    "packaging": """\
+Formal `packaging specifications`_ are maintained by the 
+`Python Packaging Authority`_ (PyPA). Refer to the PyPA website for details on
+the `specification update process`_ that must be followed when updating a 
+PyPA specification.
+
+.. _packaging specifications: https://packaging.python.org/en/latest/specifications/
+.. _Python Packaging Authority: https://www.pypa.io/en/latest/
+.. _specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
```

Comment:
Here's a suggested revision to better describe the process and the role of the PEPs within it, as well as avoid duplication.

@pradyunsg ?

```suggestion
The canonical, up-to-date packaging specifications can be found on the
`PyPA Specifications page`_ of the `Python Packaging User Guide`_
maintained by the `Python Packaging Authority`_ (PyPA).
PEPs are used to propose major additions or changes to such, and
once out of draft state are considered to be historical documents,
with any further revisions following the `PyPA specification update process`_.

.. _PyPA Specifications page: https://packaging.python.org/en/latest/specifications/
.. _Python Packaging User Guide: https://packaging.python.org/
.. _Python Packaging Authority: https://www.pypa.io/
.. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
```

Reply:
My original wording was ~30 words, the suggestion (if I've counted correctly) is close to 90 -- I hesitate to set precedent for having these additional introductions being very heavy text-wise. What are the most important additions here?

A

Reply:
I mean, the full page content (not counting the sidebar and top bar) is around 270 lines and 2000 words by my count, so I'm not sure it makes sense to worry too much over an extra 2 lines and 26 words (≈1%)...

But if we do want to make it a little shorter, this gets it down to only 15 words and one line longer than your initial draft.

```suggestion
The canonical, up-to-date packaging specifications can be found on the
`Python Packaging Authority`_ (PyPA's) `Specifications page`_.
PEPs are used to propose major additions or changes, and
once out of draft state are considered to be historical documents,
with any further revisions following the `PyPA specification update process`_.

.. _Specifications page: https://packaging.python.org/en/latest/specifications/
.. _Python Packaging Authority: https://www.pypa.io/
.. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
```

For any further additions, modifications or trimming beyond that, I would defer to @pradyunsg , who has the most expertise in the subject matter among all of us.

Reply:
I'm OK with having slightly longer text here; I imagine the packaging community would like to use this page as an intro to their PEPs process, so it's good to have some discussion of how packaging PEPs work. I like @CAM-Gerlach's second wording suggestion best among the options presented.

Reply:
I like @CAM-Gerlach's second suggestion, although I'd replace:

```
PEPs are used to propose major additions or changes, and
once out of draft state are considered to be historical documents,
with any further revisions following the `PyPA specification update process`_.
```

with

```
Packaging PEPs follow the `PyPA specification update process`_. They are used to propose major additions or changes to interoperability specifications. Once out of draft state, these PEPs are considered to be historical documents.
```